js/core.js

```diff
@@ -1913,6 +1908,55 @@ var ERMrest = (function (module) {
                     module._fixedEncodeURIComponent(this.name)].join(":");
         },
 
+        /**
+         * return the default value for a column after checking whether it's a primitive that can be displayed properly
+         * @return {string}
```

Comment:
These values are being used to display in the UI. In the end, when this data is submitted to `ermrestJS`, all of this data is in strings. The input fields will properly try to show those values in the UI.